tests/test_app/test_store_code.rs

```diff
@@ -1,3 +1,5 @@
+#![cfg(feature = "cosmwasm_1_2")]
```

Comment:
`App::store_code` should work with no features enabled. Instead of marking whole file please mark only the `store_code_generates_default_address_for_creator` test.

Reply:
Surely, before the end version is ready, all such things will be cleaned up.